packages/@uppy/dashboard/src/components/AddFiles.jsx

```diff
@@ -39,6 +47,35 @@ class AddFiles extends Component {
     )
   }
 
+  renderHiddenCameraInput = (type, refCallback) => {
+    let accept
+    switch (type) {
+      case 'photo':
+        accept = 'image/*'
+        break
+      case 'video':
+        accept = 'video/*'
+        break
+      default:
+        break
+    }
```

Comment:
```suggestion
    const typeToAccept = { photo: 'image/*', video: 'video/*' }
    const accept = typeToAccept[type]
```

Reply:
should probably document as well why `undefined` is a valid value and when happens when it is